### DISCUSSION TITLE
Video Solution For coding problems

### DISCUSSION THREAD

**Discussion by SahityaRoy** [Ayush7614/Daily-Coding-DS-ALGO-Practice#212] [Ideas]
I think it would be very helpful if we added videos for every solution .....
What do u think ??
[Demo](https://www.youtube.com/watch?v=7tncYsu_2l8&t=142s)

**Comment by UG-SEP** [comment]
> I think it would be very helpful if we added videos for every solution .....
> What do u think ??
> [Demo](https://www.youtube.com/watch?v=7tncYsu_2l8&t=142s)

you are right @SahityaRoy but much better is to create a docs folder where the algorithm details are provided
